academic/editFM.py

```diff
@@ -20,7 +20,10 @@ def load(self, file: Path):
         self.content = []
         self.path = self.base_path / file
 
-        file = open(self.path, "r").readlines()
+        if not self.path.exists():
+            self.fm = dict()
+            return
+        file = self.path.open("r").readlines()
```

Comment:
I can't see where we are closing the file we opened. Perhaps we should explicitly close it after the contents has been processed?

Also, let's read the file with UTF-8 encoding, just as we've done for the file writing in #70?

Might be worth rebasing at the same time (fortunately, should be pretty automatic this time).

Reply:
That bug already existed, but I'll open a new PR.